src/main/java/com/rybka/model/ExchangeResponse.java

```diff
@@ -25,6 +26,6 @@
     @JsonProperty("base")
     private String base;
 
-    @JsonProperty("conversion_rates")
+    @JsonAlias({"conversion_rates", "rates"})
```

Comment:
Good choice. Did you consider to create separate DTO object?

Reply:
Thanks. Yes, I did, but I wanted to use JSON feature.